LICENSE.txt

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2018, conda-forge
```

Comment:
Please do not delete this license. It is for conda-forge itself, not for the recipe you are trying to package. You need to delete the one in `recipes/swisslandstats-geopy/LICENSE.txt`.

Reply:
Sorry, my mistake. I have amended it in [a7437d6](https://github.com/conda-forge/staged-recipes/pull/12431/commits/a7437d6760b732ec5beec93f47d1407dcb8505b0). Thank you for pointing it out.